.gitattributes

```
@@ -0,0 +1,6 @@
+* text=auto
+
+# don't diff machine generated files
+dist/ -diff
```

Is this supposed to be the `build` folder?
```suggestion
build/ -diff
```